pkg/util/mount/nsenter_mount.go

```diff
@@ -235,8 +235,13 @@ func (n *NsenterMounter) MakeRShared(path string) error {
 
 func (mounter *NsenterMounter) GetFileType(pathname string) (FileType, error) {
 	var pathType FileType
-	outputBytes, err := mounter.ne.Exec("stat", []string{"-L", `--printf "%F"`, pathname}).CombinedOutput()
+	outputBytes, err := mounter.ne.Exec("stat", []string{"-L", "--printf=%F", pathname}).CombinedOutput()
 	if err != nil {
+		if strings.Contains(string(outputBytes), "No such file") {
```

Comment:
Is it possible to check for error codes instead of strings?

Reply:
@msau42 I have tried, if `nsenter` exec fails, error msg will always be like `exit status 1`
So the answer is No